src/manifest.c

```diff
@@ -589,8 +589,18 @@ struct manifest *load_mom(int version)
 	string_or_die(&filename, "%s/%i/Manifest.MoM", state_dir, version);
 	string_or_die(&url, "%s/%i/Manifest.MoM", content_url, version);
 	if (!download_and_verify_signature(url, filename)) {
-		printf("WARNING!!! FAILED TO VERIFY SIGNATURE OF Manifest.MoM\n");
+		if (!force_ignore_unverified_signature) {
+			printf("FAILED TO VERIFY SIGNATURE OF Manifest.MoM. Can proceed if"
+				"  --force-continue-with-unverified-signature used, but system"
+				" security WILL BE COMPROMISED\n");
+			return NULL;
+		} else {
+			printf("FAILED TO VERIFY SIGNATURE OF Manifest.MoM. Operation proceeding due to"
+				"  --force-continue-with-unverified-signature, but system security WILL BE"
+				" COMPROMISED\n");
+		}
```

Comment:
Ok, I'll switch back to the "MAY BE COMPROMISED"; David wanted something more severe, but I originally used 'may' as well